docs/alerts/creating-heartbeat-alerts.md

```diff
@@ -0,0 +1,57 @@
+title: Creating Heartbeat Alerts
+description: Step-by-step Heartbeat alert creation instructions 
+
+Monitoring Apps have Heartbeat Alerts as part of their settings. You can find them in the App dropdown menu in the top-left corner.
+
+![image alt text](../images/alerts/heartbeat_image_01.png)
+
+It’s possible to create Heartbeat Alerts based on any combination of tags. Heartbeat Alerts can be tied to specific services regardless of where they are running.
+
+You can specify tags to filter and group by, then create Heartbeat Alerts for services matching your criteria. This way, even if a containerized service is moved from one host to another, it will not trigger false alerts.
+
+To make this possible Sematext Agent sends tags from the Sematext Common Schema that tag-based alerts use: 
+
+- os.host
+- container.id
+- container.image.name
+- container.image.tag
+- container.name
```

Comment:
Add backticks around these to add code formatting.

os.host -> `os.host`
etc...

Reply:
done